test/statistics.jl

```diff
@@ -0,0 +1,15 @@
+include("utils.jl")
+
+@testset "Median and Mean" begin
```

Comment:
This is fine for now, but this test will eventually be replaced, since we can implement a more efficient `Sphere`-specific one.

Reply:
Of course, this was just the first that came to my mind. With the Euclidean Fallbacks, one should check those, too, with a specific one for the sphere (you mean the online one?) we should check that, too and this test could then also be done on `Hyperbolic` or `SymmetricPositiveDefinite`, a switch to Hyperbolic would be quite easy actually.